tests/test_ml.py

```diff
@@ -3,25 +3,58 @@
 from unittest import TestCase
 
 from loda.oeis import ProgramCache
-from loda.ml import program_to_tokens, tokens_to_program, merge_programs, split_program
+from loda.ml import encoding, Model
 from tests.helpers import PROGRAMS_TEST_DIR
 
 
-class MLTests(TestCase):
+class ModelTests(TestCase):
+
+    def setUp(self):
+        self.program_cache = ProgramCache(PROGRAMS_TEST_DIR)
+        self.model = Model(self.program_cache)
+
+    def test_model_data(self):
+        # Check the number of programs.
+        num_programs = len(self.program_cache.all_ids())
+        self.assertGreater(num_programs, 0)
+
+        # Check the number of operations of merged program.
+        merged = encoding.merge_programs(self.program_cache)
+        num_operations = len(merged.ops)
+        self.assertGreater(num_operations, 0)
+
+        # Check the number of tokens in the model.
+        # One operation is encoded using three tokens.
+        self.assertEqual(3 * num_operations, len(self.model.tokens))
+
+        # Check the size of the vocabulary (specific for test dataset).
+        self.assertEqual(32, len(self.model.vocab))
```

Comment:
Fragile. When there are changes to the content of the PROGRAMS_TEST_DIR, then this value will change.

Reply:
Indeed. Changed it to non-empty check.